modules/light-clients/07-tendermint/types/genesis_test.go

```diff
@@ -1,38 +1,89 @@
 package types_test
 
 import (
-	"time"
-
 	sdk "github.com/cosmos/cosmos-sdk/types"
+
 	clienttypes "github.com/cosmos/ibc-go/modules/core/02-client/types"
-	commitmenttypes "github.com/cosmos/ibc-go/modules/core/23-commitment/types"
 	"github.com/cosmos/ibc-go/modules/light-clients/07-tendermint/types"
+	ibctesting "github.com/cosmos/ibc-go/testing"
 )
 
+// expected export ordering:
+// processed height and processed time per height
+// then all iteration keys
 func (suite *TendermintTestSuite) TestExportMetadata() {
```

Comment:
This test could be better using a for loop for the updates, but it doesn't feel worth the effort. I think this covers the necessary purposes